README.md

```diff
@@ -89,6 +103,8 @@ performant in that case.
 
 But remember: no log is always the _most_ performant!
 
+Note: category config conditionals will not work with high performance logs because we would need to evaluate the function in order to check if the conditional matches, which defeats the purpose of only calling the function when necessary.
+
```

Comment:
It would be nice to get this formatted by prettier for markdown so the line breaks like other lines.